internal/backend/deployments/metrics.go

```diff
@@ -15,8 +15,8 @@ var (
 	deploymentsCreatedCounter metric.Int64Counter       = noop.Int64Counter{}
 )
 
-func initMetrics(t *telemetry.Telemetry) {
-	deploymentsActiveGauge, _ = t.NewUpDownCounter("deployments.active", "Active deployments gauge")
-	deploymentsDrainingGauge, _ = t.NewUpDownCounter("deployments.draining", "Draining deployments gauge")
-	deploymentsCreatedCounter, _ = t.NewCounter("deployments.created", "Created deployments counter")
+func initMetrics() {
+	deploymentsActiveGauge, _ = telemetry.NewUpDownCounter("deployments.active", "Active deployments gauge")
```

Comment:
I'd love to see a design document on the metrics we want to add. We should design them and not grow them organically. See also [SLO RFC](https://docs.google.com/document/d/1TowQL6u4dDR5uJmKUklsCT1EtMxHAvv8MXkSMUeu_2o/edit?tab=t.0#heading=h.15fzcfzusxa)